### ISSUE TITLE
FetchError: invalid-json

### ISSUE THREAD

**Issue by l3nnartt** [node-fetch/node-fetch#1277]
I have a Discord Bot and on my Windows 10 64x PC it works fine, but on my VM Debian 10 Server i got the following error and I dont know why.
If i run on my computer(Windows 10 VSC Debugger) the 

![image](https://user-images.githubusercontent.com/69513317/132339835-57e808bb-b2af-40a5-a09b-caa63fa8d2a0.png)

but on my VM (Debian 10)

![image](https://user-images.githubusercontent.com/69513317/132339896-ead9f75d-70f3-4edf-a35d-ed7cef487ae4.png)

tbh i dont know why it works on my pc fine and on my server i got the error with the same packages and versions

Error:
```js
/home/lennart/Discordbots/v2/node_modules/node-fetch/lib/index.js:272
                                return Body.Promise.reject(new FetchError(`invalid json response body at ${_this2.url} reason: ${err.message}`, 'invalid-json'));
                                                           ^
FetchError: invalid json response body at https://karmatop.de/addon/pxl-api.php?name=L3nnart_ reason: Unexpected token < in JSON at position 0
    at /home/lennart/Discordbots/v2/node_modules/node-fetch/lib/index.js:272:32
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  type: 'invalid-json'
}
```

**Example Code**
```js
const url = `https://karmatop.de/addon/pxl-api.php?name=` + spieler; //spieler is a variable/argument from the discord bot, for example  "L3nnart_"
            fetch(url, {
                method: "GET",
                headers: {
                    Accept: "application/json",
                },
            })
```

### Your Environment

| software         | version
| ---------------- | -------
| node-fetch       | 2.6.1
| node             | 16.8
| npm              | 7.21.0
| Operating System | Deabian/Windows

**Comment by l3nnartt** [comment]
cloudflare bot protection bruh moment

**Comment by jimmywarting** [comment]
thinking if it might be a BOM token in the beginning...?
could you check what the text / Uint8array response is?

**Comment by Champkinz** [comment]
bump

**Comment by jimmywarting** [comment]
Bumping the wrong issue... should maybe bump #541

**Comment by Champkinz** [comment]
> Bumping the wrong issue... should maybe bump #541 with a potential fix: #1482

Thank you for that.
